dracula-theme.el

```diff
@@ -396,8 +396,8 @@ read it before opening a new issue about your will.")
                ;; ivy
                (ivy-current-match
                 ,@(if dracula-alternate-mode-line-and-minibuffer
-                      (list :weight 'normal :foreground dracula-green)
-                    (list :weight 'bold :foreground dracula-pink)))
+                      (list :weight 'normal :background dracula-current :foreground dracula-green)
+                    (list :weight 'bold :background dracula-current :foreground dracula-pink)))
```

Comment:
Hi,

Can you also add `:extend t` to the list to make the background spead on the whole line instead of stopping after the last characters?

Reply:
:extend t cannot fix it,  `(setcdr (assoc t ivy-format-functions-alist) #'ivy-format-function-line)` instead.
see https://github.com/abo-abo/swiper/issues/2383#issuecomment-675445899

Reply:
Ok, I don’t use ivy, thus I didn’t know that. If this is the expected result, we should leave it as this.